.github/workflows/update_version.yml

```
@@ -44,7 +44,7 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v5
         with:
-          python-version: 3.8
+          python-version: 3.11
```

```suggestion
          python-version: '3.x'
```

This workflow isn't doing a lot, this should be better?

---

I say we do things like that in a different change, I just want this one to cleanly remove 3.8
```suggestion
          python-version: 3.12
```